commit 5e814a15ca2b9434159dddf7bbe57c002fac079b
Author: Samuel Thibault <samuel.thibault@ens-lyon.org>
Date:   Sat Jan 6 02:27:44 2018 +0100

    hurd: Add jmp_buf-macros.h
    
    * sysdeps/mach/hurd/i386/jmp_buf-macros.h: New file.

diff --git a/sysdeps/mach/hurd/i386/jmp_buf-macros.h b/sysdeps/mach/hurd/i386/jmp_buf-macros.h
new file mode 100644
index 0000000000..834f5fb243
--- /dev/null
+++ b/sysdeps/mach/hurd/i386/jmp_buf-macros.h
@@ -0,0 +1,6 @@
+#define JMP_BUF_SIZE 32
+#define SIGJMP_BUF_SIZE 32
+#define JMP_BUF_ALIGN 4
+#define SIGJMP_BUF_ALIGN 4
+#define MASK_WAS_SAVED_OFFSET 24
+#define SAVED_MASK_OFFSET 28
